src/clue.js

```diff
@@ -84,11 +84,182 @@ name: Guest House
+	age: 31,
+	image: "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+	occupation: "Actor"
+};
+
+
+var mrsPeacock = {
+	first_name: "Eleanor",
+	last_name: "Peacock",
+	color: "blue",
+	description: "She is from a wealthy family and uses her status and money to earn popularity",
+	age: 36,
+	image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+	occupation: "Socialité"
+
+};
+
+var mrMustard = {
+	first_name: "Jack",
+	last_name: "Mustard",
+	color: "yellow",
+	description: "He is a former football player who tries to get by on his former glory",
+	age: 62,
+	image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+	occupation: "Retired Football player"
+};
+
+// Weapons
+
+var rope = {name: "rope", weight: 10};
+var knife = {name: "knife", weight: 8};
+var candlestick = {name: "candlestick", weight: 2};
+var dumbbell = {name: "dumbbell", weight: 30};
+var poison = {name: "poison", weight: 2};
+var axe = {name: "axe", weight: 15};
+var bat = {name: "bat", weight: 13};
+var trophy = {name: "trophy", weight: 25};
+var pistol = {name: "pistol", weight: 20};
+
+// Rooms
+var dinRoom = {name: "Dinning Room"};
+var conservatory = {name: "Conservatory"};
+var kitchen = {name: "Kitchen"};
+var study = {name: "Study"};
+var library = {name: "Library"};
+var billRoom = {name: "Billiard Room"};
+var lounge = {name: "Lounge"};
+var ballroom ={name: "Ballroom"};
+var hall = {name: "Hall"};
+var spa = {name: "Spa"};
+var livRoom = {name: "Living Room"};
+var observatory = {name: "Observatory"};
+var theater = {name: "Theater"};
+var gHouse = {name: "Guest House"};
+var patio = {name: "Patio"};
+
+
 // Characters Collection
-var charactersArray = [];
+var charactersArray = [mrGreen, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard];
+// charactersArray.push(mrGreen)
+// charactersArray.push(drOrchid)
+// charactersArray.push(profPlum)
+// charactersArray.push(missScarlet)
+// charactersArray.push(mrsPeacock)
+// charactersArray.push(mrMustard)
+
+
+
+
 
 // Rooms' Collection
-var roomsArray = [];
+var weaponsArray = [rope, knife, candlestick, dumbbell, poison, axe, bat, trophy, pistol];
+// roomsArray.push(rope)
+// roomsArray.push(knife)
+// roomsArray.push(candlestick)
+// roomsArray.push(dumbell)
+// roomsArray.push(rope)
+// roomsArray.push(rope)
+// roomsArray.push(rope)
+// roomsArray.push(rope)
+// roomsArray.push(rope)
+
+
 
 // Weapons Collection
-var weaponsArray = [];
+var roomsArray = [dinRoom, conservatory, kitchen, study, library, billRoom, lounge, ballroom, hall, spa, livRoom, observatory, theater, gHouse, patio];
+// weaponsArray.push(dinRoom)
+// weaponsArray.push(conservatory)
+// weaponsArray.push(kitchen)
+// weaponsArray.push(study)
+// weaponsArray.push(rope)
+// weaponsArray.push(rope)
+// weaponsArray.push(rope)
+// weaponsArray.push(rope)
+// weaponsArray.push(rope)
+
+
+function randomSelector(deck) {
+	var randomIndex = Math.floor(Math.random()*deck.length);
```

Comment:
Fix indentation, I have the feeling you are using 4 spaces and not 2.

Reply:
in my text processor i had 2 spaces and it did not show like this idk :P but i'll take a look